src/ctl/src/lib.rs

```diff
@@ -49,6 +49,12 @@ enum Commands {
     Bench(BenchCommands),
     /// Commands for tracing the compute nodes
     Trace,
+    // TODO(yuhao): profile other nodes
```

Comment:
So which node will be profiled in current implementation?

Reply:
compute node

Reply:
Which compute node?

Reply:
all of them